charts/cluster-autoscaler/Chart.yaml

```diff
@@ -11,4 +11,4 @@ name: cluster-autoscaler
 sources:
   - https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
 type: application
-version: 9.30.1
+version: 9.30.2
```

Comment:
One last request actually, can we make this a minor version bump rather than a patch as we're adding functionality.

Reply:
done